### PR TITLE
fix(config): add Leviton RZI10-1L

### PR DESCRIPTION
**Pull request by jlboygenius**
Added config settings for Leviton  RZA10 switch. Same settings as RZI06, but this model handles more power.

http://communities.leviton.com/servlet/JiveServlet/previewBody/1821-102-1-2647/Spec%20RZI06%20RZI10%20RZE06%20RZM10.pdf

### CODE REVIEW

**Comment by firstof9** [comment]
edit your title to be `fix(config): add leviton RZA10`

**Comment by blhoward2** [comment]
Thanks! Please take a look at our style guide and update the file to conform. Request a review from me when complete.

**Comment by jlboygenius** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x001d/rzi10-1l.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			"productId": "0x0206"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
```
Comment (firstof9) [inline packages/config/config/devices/0x001d/rzi10-1l.json]: Change "Group 1" to "Lifeline"

File: packages/config/config/devices/0x001d/rzi10-1l.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+			"description": "Association group 1",
```
Comment (firstof9) [inline packages/config/config/devices/0x001d/rzi10-1l.json]: Change "Association group 1" to "Lifeline"

**Review by firstof9 — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x001d/rzi10-1l.json (hunk 2 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x001d/rzi10-1l.json]: Better remove it - it adds no value over the label

File: packages/config/config/devices/0x001d/rzi10-1l.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Leviton RZI10-1L
+// Dimmer switch 1000w incandescent
+{
+	"manufacturer": "Leviton",
+	"manufacturerId": "0x001d",
+	"label": "RZI10-1L",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001d/rzi10-1l.json:6]: Is this correct or the PR title?

**Review by AlCalzone — CHANGES_REQUESTED** [review]
See other comments

File: packages/config/config/devices/0x001d/rzi10-1l.json (hunk 3 above)
Comment (jlboygenius) [reply packages/config/config/devices/0x001d/rzi10-1l.json:6]: Yes. That is the manufacturer and Label.

**Comment by jlboygenius** [comment]
> See other comments

I think you fixed it yourself.  Is this complete?

File: packages/config/config/devices/0x001d/rzi10-1l.json (hunk 1 above)
Comment (firstof9) [reply packages/config/config/devices/0x001d/rzi10-1l.json]: This still needs be changed to "Lifeline"

File: packages/config/config/devices/0x001d/rzi10-1l.json (hunk 2 above)
Comment (firstof9) [reply packages/config/config/devices/0x001d/rzi10-1l.json]: Remove this line.

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
It is not, you didn't address the requested changes.

**Review by firstof9 — APPROVED** [review]
LGTM 👍

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**